client/modules/core/helpers/apps.js

```diff
@@ -1,25 +1,15 @@
-import { Reaction, Logger } from "/client/api";
-import { Packages } from "/lib/collections";
-import { Template } from "meteor/templating";
+import _ from "lodash";
+import {Reaction} from "/client/api";
+import {Packages} from "/lib/collections";
+import {Template} from "meteor/templating";
```

Comment:
do we have a rule to enforce this?     `"space_in_paren": true` in the _.jsbeautifyrc_ will add the spaces on save, but also adds in a bunch of other places.  Eslint doesn't see anything wrong here..  which would leave me back to disabling .jsbeautify.. (nobody else using this I guess?).